webui-src/app/people/people_ownids.js

```diff
@@ -1,213 +1,399 @@
-let m = require('mithril');
-let rs = require('rswebui');
-let widget = require('widgets');
-let people_util = require('people/people_util')
+let m = require("mithril");
+let rs = require("rswebui");
+let widget = require("widgets");
+let people_util = require("people/people_util");
 
+function checksudo(data) {
+  if (data === "0000000000000000") return true;
+  return false;
+}
```

Comment:
`checksudo` is a pure function so it can be moved to util file.

Reply:
Also you can write is as:
```js
return (data === "0000000000000000");
```

Reply:
Sure @rottencandy, I will put it in the poeple_util class.